### ISSUE TITLE
New feature suggestion

### ISSUE THREAD

**Issue by arulvenkatesh** [up9inc/mockintosh#145]
Hi @undera @mertyildiran,
Thanks for having such an wondeful feature of mocking rest api and kakfa producer/consumer events. I came across a sitaution in the current project that I'm working where my current application calling a rest api which sends 200 back in response and also posts an event to a kafka topic. The kafka event is also consumed back again by the application. 

I would like to suggest an new feature where we have an mock response as step 1 and publish an event to the kafka topic as a subsequent event. 

![image](https://user-images.githubusercontent.com/22679519/127862352-5c2053fb-d91a-436a-840f-ae1e73cee66a.png)

**Comment by undera** [comment]
Hi,
This was actually already implemented. Check the `triggerAsyncProducer` option for REST response on this page: https://mockintosh.io/Async.html

**Comment by undera** [comment]
Did it work for you @arulvenkatesh ? Can  I help somehow more?

**Comment by mertyildiran** [comment]
@undera I suspect that @arulvenkatesh asks for a feature that's the consumer version of `triggerAsyncProducer` such that an HTTP endpoint triggers consuming upon request. I'm afraid there is no such feature in Mockintosh right now.

Adding such a feature means that we will introduce a new field like `triggerAsyncConsumer`.

I also thought about a feature in the past that an asynchronous actor alters the state of an HTTP endpoint upon consuming or producing such that it starts to respond differently.

In conclusion; these triggers are already implemented:

- HTTP endpoint --> Async producer

and these triggers are missing right now:

- HTTP endpoint --> Async consumer
- Async producer --> HTTP endpoint
- Async consumer --> HTTP endpoint

Last two are a little bit hard to achieve since it requires stateful HTTP endpoints. While I believe we can implement the "HTTP endpoint --> Async consumer" relatively easily.

**Comment by arulvenkatesh** [comment]
Sorry guys, I’m unwell for last couple of days and couldn’t test this. I think my requirement will satisfy with http endpoint- Async Producer trigger.

Once I recover I will test and let you know.

**Comment by arulvenkatesh** [comment]
Hi there,
Today I tried testing my scenario with the docker image and I was unsuccessful. 

I get this error. Do you how can I check the logs ?

Also does it support avro messages? 




We Sent from my iPhone

**Comment by undera** [comment]
You see the log in your command-line.
It says that you need to provide Kafka to connect to (mockintosh does not provide it).
Configure `address` option pointing to your existing Kafka broker to resolve it.
